site2/docs/reference-cli-tools.md

```diff
@@ -288,7 +288,7 @@ Commands
 Options
 |Flag|Description|Default|
 |---|---|---|
-|`--auth-params`|Authentication parameters, for example key1:val1,key2:val2||
+|`--auth-params`|Authentication parameters, whose format is determined by the implementation of method `configure` in authentication plugin class, for example "key1:val1,key2:val2" or "{\"key1\":\"val1\",\"key2\":\"val2\"}"||
```

Comment:
We should also fix the same description in the JCommander help string that is shown when using the tool